gradle-plugin/settings.gradle

```diff
@@ -19,7 +19,7 @@ dependencyResolutionManagement {
       for (Map.Entry<String, String> entry : overrides.entrySet()) {
         String key = entry.getKey()
         String value = entry.getValue()
-        String catalogKey = key.substring("override_".length()).toLowerCase(Locale.US)
+        String catalogKey = key.substring("override_".length())
```

Comment:
@ZacSweers Do you remember why you did this? This was causing issues.

Reply:
I don't remember